test/controllers/HelloWorldControllerSpec.scala

```diff
@@ -35,7 +35,7 @@ class HelloWorldControllerSpec extends WordSpec with Matchers with GuiceOneAppPe
     private val configuration = Configuration.load(env)
 
     private val serviceConfig = new ServicesConfig(configuration, new RunMode(configuration, Mode.Dev))
-    private val appConfig = new AppConfig(configuration, serviceConfig)
+    private val appConfig = new AppConfigImpl(configuration, serviceConfig)
```

Comment:
Following the definition of a mock AppConfig these tests can be simplified avoiding the necessity to create the environment components required to create a real AppConfig instance. See the equivalent test in manage-external-guidance-frontend.

Reply:
This the HelloWorld test and will most likely dissappear at some point